core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -328,6 +331,8 @@ describe('Overview', () => {
       });
 
       it('Verify METRICS Navigation for Lambda' , () => {
+            browser.driver.switchTo().activeElement();
+            browser.sleep(5000);
```

Comment:
Can we use a Param instead of hard coding 5000?

Reply:
This file's been deleted